doc/BEGINNERS_TUTORIAL.org

```diff
@@ -0,0 +1,184 @@
+* Why Spacemacs?
+- Unparallelled text and structure editing for all types of writing tasks: creative writing, blogging, note-taking, todo-lists,
+  scientific papers... 
+- Powerful modes for programming in dozens of programming languages
+- Deeply customizable yet beginner-friendly
+* Installation
+ Spacemacs is a beginner-friendly and powerful distribution of a popular text
+ editor called Emacs. To install Spacemacs you need to first install base Emacs and then download
+ and install the Spacemacs extension files, which is easiest done by using a program
+ called Git. The steps are easy and outlined below.
+** 1. Install Emacs
+Follow the instructions in https://github.com/syl20bnr/spacemacs#prerequisites.
```

Comment:
I think we should point out that Windows users need Gnu-TLS support for HTTPS to work (although it's possible to use HTTP instead). The recommended 64-bit build has it (http://emacsbinw64.sourceforge.net/), but the official 32-bit build doesn't (https://www.gnu.org/software/emacs/download.html).
If necessary, Gnu-TLS can be installed from http://www.gnutls.org/download.html. To check if Emacs supports Gnu-TLS:
```
M-: (gnutls-available-p)
```